src/EnergyPlus/DataHeatBalSurface.cc

```diff
@@ -159,88 +159,88 @@ namespace DataHeatBalSurface {
+    Array1D<Real64> SurfOpaqAvgFaceCondLossRep; // Equals -Opaq Surf average Face Cond
     // when Opaq Surf average Face Cond  < 0
-    Array1D<Real64> OpaqSurfAvgFaceConduction; // Opaque surface average heat conduction flow (W)
+    Array1D<Real64> SurfOpaqAvgFaceConduction; // Opaque surface average heat conduction flow (W)
     // net conduction from outside environ toward inside zone
     //  from inside of opaque surfaces, for reporting (W)
-    Array1D<Real64> OpaqSurfAvgFaceConductionFlux; // Opaque surface average face heat conduction flux (W/m2)
+    Array1D<Real64> SurfOpaqAvgFaceConductionFlux; // Opaque surface average face heat conduction flux (W/m2)
     // net conduction from outside environ to inside zone
     //  from inside of opaque surfaces, for reporting (W/m2)
-    Array1D<Real64> OpaqSurfAvgFaceConductionEnergy; // Opaque surface average heat conduction flow (J)
+    Array1D<Real64> SurfOpaqAvgFaceConductionEnergy; // Opaque surface average heat conduction flow (J)
     // net conduction from outside environ toward inside zone
     //  from inside of opaque surfaces, for reporting (J)
 
-    Array1D<Real64> OpaqSurfStorageGainRep; // Equals Opaque surface stored heat conduction flow
+    Array1D<Real64> SurfOpaqStorageGainRep; // Equals Opaque surface stored heat conduction flow
     // when Opaque surface stored heat conduction flow  >= 0
-    Array1D<Real64> OpaqSurfStorageCondLossRep; // Equals -Opaque surface stored heat conduction flow
+    Array1D<Real64> SurfOpaqStorageCondLossRep; // Equals -Opaque surface stored heat conduction flow
     // when Opaque surface stored heat conduction flow   < 0
-    Array1D<Real64> OpaqSurfStorageConduction; // Opaque surface stored heat conduction flow (W)
+    Array1D<Real64> SurfOpaqStorageConduction; // Opaque surface stored heat conduction flow (W)
     // storage of heat inside surface, positive is increasing in surf
-    Array1D<Real64> OpaqSurfStorageConductionFlux; // Opaque surface stored heat conduction flux (W/m2)
+    Array1D<Real64> SurfOpaqStorageConductionFlux; // Opaque surface stored heat conduction flux (W/m2)
     // storage of heat inside surface, positive is increasing in surf
-    Array1D<Real64> OpaqSurfStorageConductionEnergy; // Opaque surface stored heat conduction flow (J)
+    Array1D<Real64> SurfOpaqStorageConductionEnergy; // Opaque surface stored heat conduction flow (J)
     // storage of heat inside surface, positive is increasing in surf
 
-    Array1D<Real64> OpaqSurfInsFaceBeamSolAbsorbed; // Opaque surface inside face absorbed beam solar,
+    Array1D<Real64> SurfOpaqInsFaceBeamSolAbsorbed; // Opaque surface inside face absorbed beam solar,
     // for reporting (W)
-    Array1D<Real64> TempSurfOut; // Temperature of the Outside Surface for each heat transfer surface
+    Array1D<Real64> SurfTempSurfOut; // Temperature of the Outside Surface for each heat transfer surface
```

Comment:
Minor, but names like this which already have `Surf` in them could simply be `SurfTempOut`.

Reply:
I renamed this globally.